distribution/docker/Dockerfile

```diff
@@ -18,36 +18,15 @@
 #
 
 ARG JDK_VERSION=11
+FROM alpine as extractor
 
-# The platform is explicitly specified as x64 to build the Druid distribution.
-# This is because it's not able to build the distribution on arm64 due to dependency problem of web-console. See: https://github.com/apache/druid/issues/13012
-# Since only java jars are shipped in the final image, it's OK to build the distribution on x64.
-# Once the web-console dependency problem is resolved, we can remove the --platform directive.
-FROM --platform=linux/amd64 maven:3.8.6-jdk-11-slim as builder
+ARG VERSION=
```

Comment:
Is this intentionally set to blank?

Reply:
Yeah, that's set during build time. It gets set in [distribution/pom.xml](https://github.com/confluentinc/druid/pull/136/files#diff-ac19e9d7369629bf473b9b6c514012d5fecc319f98a6b9b57855f3202e970610R795)

As a part of this commit https://github.com/confluentinc/druid/pull/136/commits/d4648d22bda94760e5f6b18f049c17e93379fabe